pallets/fragments/src/lib.rs

```diff
@@ -342,7 +342,7 @@ pub mod pallet {
 		_,
 		Identity,
 		Hash128,
-		FragmentDefinition<T::AssetId, T::AccountId, T::BlockNumber>,
+		FragmentDefinition<Vec<u8>, T::AssetId, T::AccountId, T::BlockNumber>,
```

Comment:
Not useful consider we use definition only here

Reply:
reason give above (basically because `DefinitionMetadata` is a parameter of one of the extrinsics)